app/services/project_reporter.rb

```diff
@@ -30,11 +30,11 @@ def message
   end
 
   def report_message
-    return unless description.present?
+    return if description.blank?
```

Comment:
Sorry I should have been more clear with this one

```
if description.present
     "**Reporter's message**: #{description}"
end
```

Returning early from a method isn't as readable as a condition.